### ISSUE TITLE
iOS 8 Target Depolyment Warnings

### ISSUE THREAD

**Issue by TheBearF8** [SDWebImage/SDWebImage#3097]
Showing Recent Messages
The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.0.99.

**Comment by brunodelgado** [comment]
Are you using Swift Package Manager?
This is a known warning and it'll be fixed on SDWebImage 6.0 release.

https://github.com/SDWebImage/SDWebImage/issues/3085

**Comment by kikeenrique** [comment]
Would it be possible to consider to include a Package@swift.5.3.swift modifying platforms parameter?
  ```
platforms: [
        .macOS(.v10_10),
        .iOS(.v9),
        .tvOS(.v9),
        .watchOS(.v2)
    ]

```

Regards
